can_tools/scrapers/official/OH/oh_vaccine_demographics.py

```diff
@@ -54,9 +54,11 @@ def _extract_data(self, shot_type: str) -> pd.DataFrame:
         for county in counties:
             print("working on county: ", county)
             wb = ws.select("county", county)
-            df = wb.getWorksheet("New Demographics").data.assign(location_name=county)
+            try:
+                df = wb.getWorksheet("New Demographics").data.assign(location_name=county)
+            except TypeError:
+                print(f"Could not fetch data for {county}, skipping...")
```

Comment:
Sometimes, one or two counties are not included in the data tables (I'm presuming b/c they did not report data, but not sure.) 

Currently, if any counties are missing the scraper breaks. This is great as it lets us know when any of the counties are missing, but it also means that if we're missing data for a single county then we don't get data for any counties :/ 

This lets the scraper continue without the missing counties, which I'm a little apprehensive about because we won't be alerted if more and more counties don't have data. But, I'm not sure of a great alternative. We could potentially set up a logger for the Prefect tasks but I'm not sure if that's worth the effort.

Reply:
Yeah, this seems fine for now.  This is kind of a systemic problem (the compare tool doesn't highlight when locations stop updating until they disappear completely :-/).